docs/source/config/webui.rst

```diff
@@ -47,7 +47,7 @@ On a web client side, a file ``config.js`` in a root of the project which contai
 
 Multiple servers could be configured for user to pick from. To disconnect from the current server and return to login screen, pick 'Disconnect' from the drop down at the top right corner of the UI.
 
-On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com:8000`, your config should look like that:
+On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com`, your config should look like that:
```

Comment:
@emedvedev: `allow_origin` in that context will only apply if user is using the standalone daemon. Otherwise, we set `*` at https://github.com/StackStorm/st2workroom/blob/0a96e6f811c1cb3577888af71721ae9ea887b977/modules/profile/manifests/st2server.pp#L157. 

If we find we need to make this configurable for users, we can re-address. But, the value in the config is all but ignored when served by something else.